### ISSUE TITLE
Using <Picture /> tag causes the build to silently fail on Netlify 

### ISSUE THREAD

**Issue by JayBox325** [withastro/astro#12779]
### Astro Info

```block
Astro                    v5.0.9
Node                     v18.18.2
System                   macOS (arm64)
Package Manager          npm
Output                   server
Adapter                  @astrojs/netlify
Integrations             @astrojs/react
                         astro-icon
```

### Describe the Bug

Firstly, apologies if this should be raised with Netlify, but I thought as the offending code snippet is Astro, I thought I'd start here.

I am using the Astro <Picture /> element to render imagery, but this causes the build on Netlify (works fine locally) to fail from the point of the <Picture /> usage onwards. Everything builds fine until the <Picture /> element then fails below.

There are also no warnings or errors in Netlify to indicate what the build error could be. Below is my code snippets I have tried all to the same result:

```
<Picture
        formats={['avif', 'webp', 'jpeg']}
        src={'https://cdn.sanity.io/images/4g23k0t7/production/5adfc1e1d38dbf81d8259ca3513e380659b11814-2048x1365.jpg'}
        alt=""
        height={675}
        width={1200}
        class="object-cover max-w-[102%] w-[102%] h-full"
        layout="full-width"
 />
```
        
```
<Picture
        formats={['avif', 'webp', 'jpeg']}
        src={'/corner-flag.png'}
        alt=""
        height={1750}
        width={2000}
        class="object-cover max-w-[102%] w-[102%] h-full"
/>
```       
 
```
<Picture
        formats={['avif', 'webp', 'jpeg']}
        src={'https://cdn.sanity.io/images/4g23k0t7/production/5adfc1e1d38dbf81d8259ca3513e380659b11814-2048x1365.jpg'}
        alt=""
        height={1750}
        width={2000}
        class="object-cover max-w-[102%] w-[102%] h-full"
        layout="responsive"
/>
```

To confirm, `cdn.sanity.io` is in my approved image domains, I have tried it with other external image URLS (`https://theguardian.engineering/_astro/home-digital-development.QXP0mqox_Z3Yfhy.webp` was my chosen external example from the Astro site.)

### What's the expected result?

I'd expect the <Picture /> element to work on a production Netlify build as it does in my local builds.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-pqbgbmfp?file=src%2Fpages%2Findex.astro

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by ascorbic** [comment]
I'm not sure of the exact issue, but can I suggest a solution for the immediate case? Netlify supports content negotiation, so will send the best format supported by the user's browser. Instead of Picture, just use Image and it will still send the right format.

**Comment by JayBox325** [comment]
Yeah, I'm happy to switch to the <Image /> element for now, but when I use the <Picture /> element locally I get much better results, as the Picture element can return a 136kb .heif image file, while the Image element returns a 249kb .avif image file. Which is over 100kb that I could be saving if the Picture element was working with Netlify.

Would it be worth raising this with Netlify or is keeping this ticket open here the best approach to getting this resolved?

**Comment by JayBox325** [comment]
Happy to close.
